
real	0m40.089s
user	0m35.349s
sys	0m1.847s

real	1m57.853s
user	1m50.999s
sys	0m2.474s

real	1m27.177s
user	1m12.575s
sys	0m8.360s

real	4m25.087s
user	3m39.365s
sys	0m25.583s
Traceback (most recent call last):
  File "/home/thiago.esterci/2D_imune_edema_pinn/edo_pinn_model.py", line 187, in <module>
    Cl = pk.load(f)
EOFError: Ran out of input

real	0m2.718s
user	0m1.862s
sys	0m0.266s

real	3m6.453s
user	2m55.066s
sys	0m3.052s

real	1m37.864s
user	1m31.640s
sys	0m2.201s

real	3m48.437s
user	3m22.917s
sys	0m13.432s

real	1m39.174s
user	1m22.577s
sys	0m8.518s

real	2m40.909s
user	2m30.879s
sys	0m3.832s

real	1m4.731s
user	0m58.907s
sys	0m2.018s

real	3m4.188s
user	2m55.102s
sys	0m2.902s

real	1m49.147s
user	1m38.082s
sys	0m4.721s

real	2m57.571s
user	2m48.251s
sys	0m2.976s

real	1m23.500s
user	1m14.587s
sys	0m4.058s

real	2m37.845s
user	2m28.748s
sys	0m2.911s

real	1m3.001s
user	0m58.499s
sys	0m1.817s

real	2m40.997s
user	2m33.157s
sys	0m2.764s

real	2m5.658s
user	1m51.933s
sys	0m8.922s

real	3m6.874s
user	2m59.175s
sys	0m2.631s

real	0m52.656s
user	0m49.151s
sys	0m1.582s

real	2m22.035s
user	2m16.471s
sys	0m2.286s

real	1m4.753s
user	0m59.070s
sys	0m3.634s

real	3m7.740s
user	2m48.629s
sys	0m14.484s

real	0m35.464s
user	0m31.672s
sys	0m2.300s

real	1m36.374s
user	1m25.280s
sys	0m8.237s

real	0m33.820s
user	0m30.802s
sys	0m1.661s

real	1m37.766s
user	1m29.254s
sys	0m6.083s

real	0m36.556s
user	0m32.496s
sys	0m2.656s

real	1m32.452s
user	1m28.731s
sys	0m1.789s

real	0m33.712s
user	0m30.289s
sys	0m1.943s

real	1m31.803s
user	1m23.323s
sys	0m5.906s

real	0m36.561s
user	0m32.228s
sys	0m2.760s

real	1m28.455s
user	1m24.426s
sys	0m1.913s

real	0m33.404s
user	0m30.846s
sys	0m1.273s

real	1m30.967s
user	1m27.078s
sys	0m1.879s

real	0m34.994s
user	0m31.386s
sys	0m2.061s

real	1m36.176s
user	1m25.731s
sys	0m7.891s

real	0m30.932s
user	0m27.722s
sys	0m1.892s

real	1m17.903s
user	1m14.600s
sys	0m1.660s
